scrapers/AdultimeAPI.py

```diff
@@ -370,12 +382,38 @@ def create_index(api_json):
     index_value = "{}|{}|{}|{}{}{}{}".format(json_id, json_title, json_length, json_size_4k, json_size_1080, json_size_720, json_size_480)
     return index_value
 
+def callGraphQL(query, variables=None):
+    json = {'query': query}
+    if variables is not None:
+        json['variables'] = variables
+    response = requests.post(SERVER_URL, json=json, headers=STASH_HEADERS)
+    if response.status_code == 200:
+        result = response.json()
+        if result.get("error"):
+            for error in result["error"]["errors"]:
+                raise Exception("GraphQL error: {}".format(error))
+        if result.get("data"):
+            return result.get("data")
+    elif response.status_code == 401:
+        sys.exit("HTTP Error 401, Unauthorised.")
+    else:
+        raise ConnectionError("GraphQL query failed:{} - {}. Query: {}. Variables: {}".format(
+            response.status_code, response.content, query, variables))
+
+def getdbPath():
+    query = "query Configuration {  configuration {...ConfigData}}fragment ConfigData on ConfigResult {  general {...ConfigGeneralData}}fragment ConfigGeneralData on ConfigGeneralResult { databasePath }"
+    result = callGraphQL(query)
+    return result["configuration"]["general"]["databasePath"]
+
 
 fragment = json.loads(sys.stdin.read())
 
 # Get your database
 DB_PATH = None
```

Comment:
DB_PATH if fetched from getdbPatgh below, shouldnt this be removed? (not needed)

Reply:
I guess it don't harm anyone with this, if the user is too lazy to put the ApiKeys, this method could still work